osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -201,13 +201,14 @@ private Cipher getCipherForDecryption()
         return cipher;
     }
 
-    private KeyPair createKeyPair(final String rsaPublicKeyPath, final String rsaPrivateKeyPath) {
+    private KeyPair createKeyPair(final String rsaPublicKeyPath, final String rsaPrivateKeyPath)
+            throws EncrypterException {
         final PrivateKey privateKey = readPrivateKeyFromFile(rsaPrivateKeyPath);
         final PublicKey publicKey = readPublicKeyFromFile(rsaPublicKeyPath);
         return this.createKeyPair(publicKey, privateKey);
     }
 
-    private KeyPair createKeyPair(final PublicKey publicKey, final PrivateKey privateKey) {
+    private KeyPair createKeyPair(final PublicKey publicKey, final PrivateKey privateKey) throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)